### DISCUSSION TITLE
Language management for v6

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#533] [Ideas]
I don't know if you have thought of improving the management of multilingual fields in the next V6, but what do you think of this approach?
<img width="1544" alt="BKP-new-idea-ML-flat2" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/4eaf3b8a-b465-4d89-b95f-6fcfb93d16d3">

**Comment by blondie63** [comment]
@o15a3d4l11s2 what you about this idea ?

**Reply by o15a3d4l11s2** [reply]
In regard to the UI your proposal makes editing easier. But I think programming-wise you solution would need big code changes to make it possible. I do not use the translations feature very often, so I would have to create a new test and get back with a more specific answer.

**Comment by Yuutakasan** [comment]
Since adding records degrades performance, multilingualization is easier to manage if it can be registered as a json type.
